cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -155,3 +156,12 @@ func validateConfig(config *RunnerScaleSetListenerConfig) error {
 
 	return nil
 }
+
+func newActionsClientFromConfig(config RunnerScaleSetListenerConfig, creds *actions.ActionsAuth, options ...actions.ClientOption) (*actions.Client, error) {
```

Comment:
This isn't of much use as of yet, but makes the creation of the client testable, and it's reused in https://github.com/actions/actions-runner-controller/pull/2268